docs/source/datasets/index.rst

```diff
@@ -461,4 +461,148 @@ The following columns are included in these datasets:
 Tax form: 1040
 --------------
 As with data collected from W-2 and 1099 forms, Pseudopeople will also enable the simulation of administrative records from 1040 forms, which are
-also reported to the IRS on an annual basis. This feature has not yet been implemented, so please stay tuned for more information! 
+also reported to the IRS on an annual basis. To find out more about the 1040 tax form, visit the `IRS information page <https://www.irs.gov/instructions/i1040gi>`_.
+
+Generate 1040 data with pseudopeople.generate_taxes_1040.
```

Comment:
```suggestion
Generate 1040 data with  :func:`pseudopeople.generate_taxes_1040`.
```

Reply:
This is what I had originally, but I think bc the docstring hasn't been created yet, git flagged it and failed the check

Reply:
Maybe we should add a to-do in the docs, or at least a JIRA ticket, to update this once the code is written